data_structures/binary_search_tree/test_bst.py

```diff
@@ -0,0 +1,52 @@
+def test_empty_bst_length(new_bst):
+    """ test that root is non with empty tree """
+    assert new_bst.root is None
+
+
+def test_empty_bst_insert(new_bst):
+    """ test that not empty after insert """
+    new_bst.insert(1)
+    assert new_bst.root is not None
```

Comment:
You might also want to verify that the new `root` has the value that you inserted.